TACC/Core-CMS
# redirect admin login view to use the base portal login

## Overview
Replaces the admin login view with a redirect to the base `/login`. This prevents users from seeing a login screen that isn't wired up to Tapis/LDAP.

## Testing

1. go to `<portal URL>/admin` while unauthenticated
2. Enter your credentials and check that you're redirected to the admin screen.
